### PR TITLE
Add simple runbooks for all enabled alerts

### PR DESCRIPTION
**Pull request by simu**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues.

### CODE REVIEW

**Comment by simu** [comment]
Note: some alerts don't have debugging steps, because we don't have any experience with them yet, or because we're going to link to the runbooks for replacing a disk/storage node instead.

**Review by tobru — APPROVED** [review]

File: component/alertrules.libsonnet [hunk 1]
```diff
@@ -64,6 +64,9 @@ local patch_alerts = {
   },
 };
 
+local runbook(alertname) =
+  'https://hub.syn.tools/rook-ceph/runbooks/%s.html' % alertname;
+
 /* FROM HERE: should be provided as library function by
```
Comment (corvus-ch) [inline component/alertrules.libsonnet:70]: Is/will be there an ticket to have this converted to a library?

File: docs/modules/ROOT/pages/runbooks/CephOSDSlowOps.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+= Alert rule: CephOSDSlowOps
+
+== icon:glasses[] Overview
+
+This alert fires if some Ceph OSD requests take too long to process.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/runbooks/CephOSDSlowOps.adoc:5]: Whats the concequence of this? Do IO on volumes drop or something else?

File: docs/modules/ROOT/pages/runbooks/CephPGRepairTakingTooLong.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
+= Alert rule: CephPGRepairTakingTooLong
+
+== icon:glasses[] Overview
+
+This alert fires when Ceph takes longer than one hour to repair a inconsistent placement group (PG).
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/runbooks/CephPGRepairTakingTooLong.adoc:5]:
```suggestion
This alert fires when Ceph takes longer than one hour to repair an inconsistent placement group (PG).
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]
I realy like what you have done with all those partials. It might be worth to add another one that gets included into all runbooks: "If you gain knowledge beyond what is written here, please update" or something along that line.

Also, what is the deal with all those `// Add detailed steps to debug and resolve the issue` lines?

File: component/alertrules.libsonnet (hunk 1 above)
Comment (simu) [reply component/alertrules.libsonnet:70]: https://github.com/appuio/component-openshift4-monitoring/issues/36

File: docs/modules/ROOT/pages/runbooks/CephOSDSlowOps.adoc (hunk 2 above)
Comment (simu) [reply docs/modules/ROOT/pages/runbooks/CephOSDSlowOps.adoc:5]: Expanded the description for the alert

**Comment by simu** [comment]
> I realy like what you have done with all those partials. It might be worth to add another one that gets included into all runbooks: "If you gain knowledge beyond what is written here, please update" or something along that line.

Added a note on all pages in the latest version.

> Also, what is the deal with all those `// Add detailed steps to debug and resolve the issue` lines?

The comments are part of the alert runbook template in `docs/modules/ROOT/pages/runbooks/_RuleTemplate.adoc`. I've now added a how-to describing how a new alert runbook can be generated from the template.

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**